plugins/inputs/redis/redis.go

```diff
@@ -18,7 +19,8 @@ import (
 )
 
 type Redis struct {
-	Servers []string
+	Servers   []string
+	Passwords []string
```

Comment:
Just do a single password, if the password is different across servers you will need to do multiple plugin definitions.